pan-domain-auth-verification/src/test/scala/com/gu/pandomainauth/service/CookieUtilsTest.scala

```diff
@@ -22,7 +23,7 @@ class CookieUtilsTest extends AnyFreeSpec with Matchers with EitherValues with O
       "another"),
     // The expiry is serialised to millisecond accuracy
     // so this needs to be at the same precision for comparison.
-    now().plusMillis(86400).truncatedTo(MILLIS),
+    now().plus(ofHours(1)).truncatedTo(MILLIS),
```

Comment:
The original value was 86.4 seconds, but was almost certainly *intended* to be 24 hours (86,400 seconds).

However, since the default cookie expiry is one hour, it makes more sense to use 1 hour.